README.md

```diff
@@ -1,42 +1,91 @@
 
-Download the archive for your platform from the latest [release](https://github.com/poanetwork/poa-ballot-stats/releases) and unpack it. Run the tool with `./poa-ballot-stats <options>`.
+Ballot tracking provides transparency for POA token holders and promotes validator accountability. The poa-ballot-stats tool displays active validator voting participation ordered by percentage of missed votes. 
 
-You can view the command line options with `-h`, and specify a different endpoint if your node e.g.
-uses a non-standard port. By default, it tries to connect to a local node `http://127.0.0.1:8545`.
+The default display includes:
+* non-participation/associated ballots
+* missed %
+* truncated voting key
+* truncated mining key
+* first name last name
 
-In verbose mode, with `-v`, the list of collected ballot and key change events is displayed, and for each ballot the list of participating and abstaining voters.
+![Screenshot](screenshot.png)
 
-The `-c` option takes a map with the POA contracts' addresses in JSON format. You can find the 
-current maps for the main and test network the `contracts` folder. By default, it uses `core.json`,
-for the main network.
 
-The `-p` option takes a time interval in hours, days, months, etc. E.g. `-p "10 weeks"` will only count participation in ballots that were created within the last 10 weeks. Alternatively, instead of a _time_, you can specify the earliest block _number_ as a decimal integer with the `-b` option.
+## Dependencies
+Prior to downloading poa-ballot-stats, install and activate a fully synchronized node connected to the POA network. See the [POA installation guide](https://github.com/poanetwork/wiki/wiki/POA-Installation) for instructions.
 
-Examples:
+**Note:** poa-ballot-stats must access the full network logs. Use these flags when running a node `--pruning=archive --no-warp`
+
+**Example:**
+```bash
+$ parity --chain c:\path\to\spec.json --reserved-peers c:\path\to\bootnodes.txt --pruning=archive --no-warp
+```
+
+## Installation
+
+### Stable Release
+
+Download the archive for your platform from the latest [release](https://github.com/poanetwork/poa-ballot-stats/releases) and unpack. Run the tool with `./poa-ballot-stats <options>`.
+
+#### Options
+
+`-h, --help` view command line options and help information. 
+
+`<url>` specify a different endpoint if your node uses a non-standard port. The default connects to a local node `http://127.0.0.1:8545`.
+
+`-V, --version` prints version information.
+
+
+`-v, --verbose` display collected ballot and key change events and the list of participating and abstaining voters for each ballot.
+
+`-c, --contracts <contracts>`  append a map with POA contract addresses in JSON format. The current maps for the main and test network are in the `contracts` folder. Default is the main network `core.json` file.
```

Comment:
Maybe "a map file". "A map" could be misunderstood: one might read it as a string with the values that is given explicitly on the command line.